custom/conf/app.ini.sample

```diff
@@ -286,10 +286,12 @@ LOG_SQL = true
 DB_RETRIES = 10
 ; Backoff time per DB retry (time.Duration)
 DB_RETRY_BACKOFF = 3s
-; Max idle database connections on connnection pool, default is 0
-MAX_IDLE_CONNS = 0
-; Database connection max life time, default is 3s
+; Max idle database connections on connnection pool, default is 2
+MAX_IDLE_CONNS = 2
+; Database connection max life time, default is 0 or 3s mysql
```

Comment:
comment mysql

Reply:
So we currently have a different value on MYSQL compared to all the other DBs.

I think 3s is too low and a default of 15m for all DBs could be appropriate.

Reply:
See #6804

Reply:
And #7071

Reply:
```suggestion
; Database connection max life time, default is 0 or 3s mysql (See #6804 & #7071 for reasoning)
```